src/style.css

```diff
@@ -0,0 +1,116 @@
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 300;
+  src: url("fonts/Avenir-Book.ttf");
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-style: normal;
+  font-weight: 900;
+  src: url("fonts/Avenir-Heavy.ttf");
+}
+
+html {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+}
+
+.searchBar-1 {
+  margin-top: 20px;
+  margin-bottom: 20px;
+  position: relative;
+  display: block;
+}
+
+.search-1 {
+  width: 100%;
+  position: relative;
+  padding-left: 62px;
+  height: 70px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
```

Comment:
Try to use HES instead of RGBA

Reply:
Maybe you meant HEX?
Don't think that it's a good idea, because:
1. The designer gave us color in RGBA format
2. If You do not agree, tell me how to translate colors RGBA in HEX 
3. Tried this one: https://simplecss.eu/rgbatohex.html =>working not good

Thank you!)

Reply:
UPD: Did it, but it is the designer`s responsibility